cli/cluster/instances.go

```diff
@@ -102,13 +102,22 @@ func GetInstancesConf(ctx *context.Ctx) (*InstancesConf, error) {
 	return &instancesConf, nil
 }
 
+// Removes non-local members from instances map
```

Comment:
https://go.dev/blog/godoc 
`Notice this comment is a complete sentence that begins with the name of the element it describes.`